packages/core/src/components/html-select/htmlSelect.tsx

```diff
@@ -91,7 +91,14 @@ export class HTMLSelect extends AbstractPureComponent2<HTMLSelectProps> {
 
         const optionChildren = options.map(option => {
             const props: OptionProps = typeof option === "object" ? option : { value: option };
-            return <option {...props} key={props.value} children={props.label || props.value} />;
+            return (
+                <option
+                    {...props}
+                    key={props.value}
+                    children={props.label || props.value}
+                    selected={this.props.value === props.value}
```

Comment:
I'm getting a warning (in Chrome) after this change 
This warning is the the following:

`Warning: Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>.`

I believe this line causes it.

Reply:
oops, sorry about that @dimitrisx.

@bvandercar-vt can you fix this please?

Reply:
Thanks!